torch/_inductor/__init__.py

```diff
@@ -100,9 +131,52 @@ def aoti_compile_and_package(
 
     res = package_aoti(package_path, aoti_files)
     assert res == package_path
+
+    if load_and_run:
+        compiled_model = aoti_load_package(package_path)
+        aoti_result = compiled_model(*args)
```

Comment:
We could compare with eager result here and thus enable accuracy minifier. Ok to do it in a follow-up PR.

Reply:
> We could compare with eager result here and thus enable accuracy minifier. Ok to do it in a follow-up PR.

yeah, I can do it in a follow up